test/frontend/ProfileAdminView.test.js

```diff
@@ -0,0 +1,108 @@
+/* global describe it */
+import 'jsdom-global/register'
+import React from 'react'
+import Enzyme, { mount } from 'enzyme'
+import { expect } from 'chai'
+import 'regenerator-runtime/runtime'
+
+import AdminViewEditor from '../../src/Pages/Profile/admin/AdminView'
+import AdminEditorForm from '../../src/Pages/Profile/admin/EditorForm'
+import AdminProfileEditor from '../../src/Pages/Profile/admin/Profile'
+import Adapter from 'enzyme-adapter-react-16'
+import { Badge, Container, Modal, Button, FormGroup } from 'reactstrap'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+const user = {
+  accessLevel: 2,
+  firstName: 'First',
+  lastName: 'Last',
+  middleInitial: 'I',
+  joinDate: '12-32-2012htzs342',
+  membershipValidUntil: '12-32-2012htzs342',
+  email: 'email',
+  major: 'major',
+  pagesPrinted: 20
+}
+
+describe('<AdminViewEditor />', () => {
```

Comment:
Split each `describe` to have their own files. This makes tracking down errors easier

e.g.
1. `AdminView.test.js`
2. `EditorForm.test.js`
3. `Profile.test.js`